suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx

```diff
@@ -57,14 +58,17 @@ export const PortfolioGraphHeader = () => {
     const firstGraphPoint = useAtomValue(referencePointAtom);
     const isDiscoveryActive = useSelector(selectIsDeviceDiscoveryActive);
     const isDeviceAuthorized = useSelector(selectIsDeviceAuthorized);
+    const [isUsbDeviceConnectFeatureEnabled] = useFeatureFlag(FeatureFlag.IsDeviceConnectEnabled);
     const isLoading = isDiscoveryActive || !isDeviceAuthorized;
 
     return (
         <Box>
             <VStack spacing="extraSmall" alignItems="center">
-                <Text color="textSubdued" variant="hint">
-                    <Translation id="moduleHome.graph.title" />
-                </Text>
+                {!isUsbDeviceConnectFeatureEnabled && (
```

Comment:
I think this won't be correct for:
- usb connection enabled and selected device is portfolio tracker

Reply:
Altough, after reading the task description, it's not clear what's correct behaviour to me. Could you please double-check with @shenkys?

Reply:
Hmm. Yah, it can be rad two ways. i'll check, good catch.

Reply:
I've clarified with @shenkys  and the result is: Remove that title for good.

Reply:
rebased and changed

Reply:
[b2a992d](https://github.com/trezor/trezor-suite/pull/13070/commits/b2a992d41fdfbdf3a90ac933bfc93466942a9ffa)